### ISSUE TITLE
pushState and replaceState with browser back button

### ISSUE THREAD

**Issue by evandiamond** [fancyapps/fancybox#2517]
Hi @fancyapps and thank you so much for your plugin, it's really fantastic! I had a question in regards to pushState and replaceState. Right now I am changing the URL when the fancybox is open and that URL will serve as a link to the actual file/page if a user shared that link. Next/Previous buttons change everything as expected as well. When a user closes, it goes back to the original URL. Everything so far is great 👍 

The problem I am having is, when a user uses the browser back button, it cycles through all those previous URL's in the browser address bar. Is there a way to make it so on browser back, I send the user to the original of the page they are currently on before the pushState took effect?

```
let original = window.location.href;

beforeShow: function (instance, current) {
            var hashy = $(current.opts.$orig).data('hashy');

            // Push State
            window.history.pushState('', '', hashy);
        },
        afterClose: function () {
            window.history.replaceState("", "", original);
        },
```

So basically here's the structure of what's happening, just in case I've confused you.

`Original Page` = `https://www.fancypage.com/the-gallery`

Fancybox opens user cycles thru 3 pictures, the pushState goes to:
`https://www.fancypage.com/the-gallery/image123`
`https://www.fancypage.com/the-gallery/image456`
`https://www.fancypage.com/the-gallery/image789`

The user hits back on the browser and we should bring them back to:
`https://www.fancypage.com/the-gallery` not `https://www.fancypage.com/the-gallery/image456`
